<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgDllList</class>
 <widget class="QDialog" name="DlgDllList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>进程列表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditorFilterByProcName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>&lt;通过DLL名称筛选&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="DllModuleTable" name="dllTableView"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>刷新dll列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUninstall">
        <property name="text">
         <string>卸载选中DLL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DllModuleTable</class>
   <extends>QTableView</extends>
   <header>dllmoduletable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
